src/auth/models/index.js

```diff
@@ -0,0 +1,13 @@
+'use strict';
+
+const userModel = require('./users.js');
+const { Sequelize, DataTypes } = require('sequelize');
+
+const DATABASE_URL = process.env.DATABASE_URL || 'sqlite:memory;';
```

Comment:
This is where the weird `;` memory file is coming from. The trailing semicolon is supposed to be a colon. The file it creates, `memory` should then be added to your `.gitignore`